### PR TITLE
Problem: use prebuilt image in testground

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- build image manually, don't use testground builders at all, and don't need the custom nix builder at all.
- update doc.

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

## Summary by CodeRabbit

- **Documentation**
  - Updated build image instructions and prerequisites in the README.
  - Added steps for building and loading images locally using `nix`.
  - Revised instructions for running the Testground daemon and test plans.

- **Bug Fixes**
  - Filtered out `<jemalloc>:` warning messages from command output in `cli.py`.

- **Refactor**
  - Changed builder configuration in `manifest.toml` from `docker:nix` to `docker:go`.
  - Removed and replaced builder fields in `local.toml` with artifact fields for `validators` and `fullnodes`.

### CODE REVIEW

File: testground/README.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
-# Guide
-
 [Testground documentation](https://docs.testground.ai/)
 
-## Getting started
+## Build Image
+
+>  Prerequisites: nix, linux remote builder for macOS
+
+You can test with the prebuilt images in github registry, to build the images locally for yourself:
```
Comment (yihuang) [inline testground/README.md:7]:
```suggestion
You can test with the prebuilt images in [github registry](https://github.com/crypto-org-chain/cronos/pkgs/container/cronos-testground), or build the images locally for yourself:
```

File: testground/benchmark/manifest.toml [hunk 2]
```diff
@@ -1,12 +1,9 @@
 name = "benchmark"
 
 [defaults]
-builder = "docker:nix"
+builder = "local:docker"
```
Comment (yihuang) [inline testground/benchmark/manifest.toml:4]:
```suggestion
builder = "docker:go"
```

File: testground/README.md [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
+## Build Image
+
+>  Prerequisites: nix, linux remote builder for macOS
+
+You can test with the prebuilt images in [github registry](https://github.com/crypto-org-chain/cronos/pkgs/container/cronos-testground), or build the image locally:
 
-### Prerequisites
+```bash
+$ nix build .#testground-image
+# for mac: nix build .#legacyPackages.aarch64-linux.testground-image
```
Comment (mmsqe) [inline testground/README.md:11]: do we need link https://nixos.org/manual/nixpkgs/unstable/#sec-darwin-builder

File: testground/README.md [hunk 4]
```diff
@@ -1,13 +1,20 @@
-# Guide
-
 [Testground documentation](https://docs.testground.ai/)
 
-## Getting started
+## Build Image
+
+>  Prerequisites: nix, linux remote builder for macOS
```
Comment (yihuang) [inline testground/README.md:5]:
```suggestion
>  Prerequisites: nix, for macOS also need [linux remote builder](https://nix.dev/manual/nix/2.22/advanced-topics/distributed-builds.html)
```

File: testground/README.md (hunk 3 above)
Comment (yihuang) [reply testground/README.md:11]: that's one of the ways to run a remote builder.

**Review by mmsqe — APPROVED** [review]

**Pull request merged**